synapse/config/captcha.py

```diff
@@ -23,21 +23,45 @@
 
 from synapse.types import JsonDict
 
-from ._base import Config, ConfigError
+from ._base import Config, ConfigError, read_file
+
+CONFLICTING_RECAPTCHA_PRIVATE_KEY_OPTS_ERROR = """\
+You have configured both `recaptcha_private_key` and
+`recaptcha_private_key_path`. These are mutually incompatible.
```

Comment:
We can dumb this down a bit:

```suggestion
You have configured both `recaptcha_private_key` and
`recaptcha_private_key_path`. Only one or the other can be specified at a time.
```

Reply:
> For these PRs I did not use my own wording, but took it from `registration_shared_secret`: https://github.com/element-hq/synapse/blob/aab3672037ee9ea71c54c89c86d9c0cc35ae184f/synapse/config/registration.py#L37-L40

Fair enough :fast_forward: 

It looks like we aren't consistent with this either. Another version:

https://github.com/element-hq/synapse/blob/51296684498012217d6104deb5509f15dad93864/synapse/config/key.py#L94-L97